packages/runtime/container-runtime/src/summary/summarizer.ts

```diff
@@ -40,17 +38,16 @@ import {
 	ISummarizerRuntime,
 	ISummarizingWarning,
 	type IRetriableFailureError,
+	type ISummaryConfiguration,
 } from "./summarizerTypes.js";
 import { SummaryCollection } from "./summaryCollection.js";
 import { SummarizeResultBuilder } from "./summaryGenerator.js";
 
-const summarizingError = "summarizingError";
-
 export class SummarizingWarning
 	extends LoggingError
 	implements ISummarizingWarning, IFluidErrorBase
 {
-	readonly errorType = summarizingError;
+	readonly errorType = "summarizingError";
```

Comment:
The const is redundant; this won't compile with any other value due to implementing `ISummarizingWarning`.